src/lib/mina_caqti/mina_caqti.ml

```diff
@@ -0,0 +1,9 @@
+(* mina_caqti.ml -- Mina helpers for the Caqti database bindings *)
```

Comment:
I wonder what's the point of introducing this helper library instead of rewriting the whole code to the new syntax introduced by Caqti?

Reply:
sad to admit that it simply made the find-and-replace regex job easier

Reply:
Do you support an idea of doing one more find-and-replace to remove these aliases?